README.md

```diff
@@ -263,7 +263,8 @@ GRIST_FORCE_LOGIN    | Much like GRIST_ANON_PLAYGROUND but don't support anonymo
 GRIST_SINGLE_ORG | set to an org "domain" to pin client to that org
 GRIST_TEMPLATE_ORG | set to an org "domain" to show public docs from that org
 GRIST_HELP_CENTER | set the help center link ref
-FREE_COACHING_CALL_URL | set the link to the human help (example: email or meeting scheduling tool)
+FREE_COACHING_CALL_URL | set the link to the human help (example: email adress or meeting scheduling tool)
+CONTACT_SUPPORT_URL | set the link to contact support on error pages (example: email adress or online form)
```

Comment:
```suggestion
GRIST_CONTACT_SUPPORT_URL | set the link to contact support on error pages (example: email adress or online form)
```

Nitpicking here, but prefixing is a little bit safer (wish I remembered to suggest it for `FREE_COACHING_CALL_URL` before it was added).